### PR TITLE
zathura-pdf-mupdf: new package at 0.3.5

### PR DESCRIPTION
**Pull request by ioraff**
Read: <getkiss.org/guidestones.txt>

## Description of package
mupdf frontend to zathura

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/zathura-pdf-mupdf/depends [hunk 1]
```diff
@@ -0,0 +1,9 @@
+cairo
+jbig2dec
+libjpeg-turbo
+libmupdf make
+libressl
+meson make
+openjpeg2
+samurai make
```
Comment (dylanaraps) [inline community/zathura-pdf-mupdf/depends]: This can be removed.

File: community/zathura-pdf-mupdf/build [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+meson build \
+    --prefix=/usr \
+    -Dlink-external=true \
+
+ninja -C build
+DESTDIR="$1" ninja -C build install
```
Comment (dylanaraps) [inline community/zathura-pdf-mupdf/build]: See: 

```sh
#!/bin/sh -e

export DESTDIR="$1"

meson \
    --prefix=/usr \
    --sysconfdir=/etc \
    --mandir=/usr/share/man \
    -Ddocs=false \
    -Dintrospection=false \
    . output

ninja -C output
ninja -C output install
```

This is how we typically format `meson` builds.

File: community/zathura-pdf-mupdf/depends [hunk 3]
```diff
@@ -0,0 +1,8 @@
+cairo
+jbig2dec
+libjpeg-turbo
+libmupdf make
```
Comment (dylanaraps) [inline community/zathura-pdf-mupdf/depends]: Only a `make` dep?

Comment (ioraff) [reply community/zathura-pdf-mupdf/depends]: Yes. As far as libraries go, `libmupdf` only contains object archives and headers. I've tested this.

Comment (dylanaraps) [reply community/zathura-pdf-mupdf/depends]: :+1:

**Pull request merged**